config/installers/rolebasedcreds/namespaced/kustomization.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Adds namespace to all resources.
+namespace: PLACEHOLDER-NAMESPACE
+namePrefix: sagemaker-k8s-operator-
+  
+bases:
+- ../../../base
+- resources
+
+resources:
+- ../../../rbac/namespaced
+
+patchesStrategicMerge:
+- resources/manager_add_service_account_patch.yaml
+
+patchesJson6902:
+# Rename created namespace
+- target:
+    version: v1
+    kind: Namespace
+    name: sagemaker-k8s-operator-system
```

Comment:
Q: So is this the default namespace and gets overwritten if user specifies it ?

Reply:
This block fetches the namespace that is created by the default Kustomize target and renames it to "PLACEHOLDER-NAMESPACE". We are planning on creating a script that replaces all instances of "PLACEHOLDER-NAMESPACE" with the desired namespace of the installation.

Reply:
Where is default kustomize target ?My question is more towards what happens if user does not specifies the namespace?

Reply:
The user never needs to "specify" a namespace. They either use the cluster-spaced installer that we have provided previously, or they can modify this namespaced installer (replacing the string) to install into their specific namespace. If they install this without modifying it, it will throw an error because a namespace must have all lowercase letters.

Reply:
> Where is default kustomize target ?

`config/installers/rolebasedcreds/kustomization.yaml`